content/learn/userguides/secondary-analysis/10x.md

```diff
@@ -4,8 +4,28 @@ date: "2018-05-03"
 title: "Optimus Prime Workflow in Detail"
 ---
 
-## Description of 3-prime scRNA sequencing
+## Description of droplet-based 3-prime scRNA sequencing 
 
+The introduction of droplet-based technologies such as inDrop ([Klein, et al., 2015](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4441768/))and Drop-seq ([Macosko, et al., 2015](https://www.sciencedirect.com/science/article/pii/S0092867415005498)) moved the throughput of a single-cell RNA sequencing experiment from hundreds, to thousands, of cells. Current droplet technologies combine 1) the use of microfluidics to capture individual cells in oil droplets containing barcoded beads, 2) short read 3’ single-strand sequencing, and 3) tagging of each transcript with a unique molecular identifier (UMI) as well as a cell barcode. These innovations enable every transcript that is captured from individual cells to be mapped to its cognate cell and counted as a single transcript, correcting for PCR bias as well as batch effects. 
```

Comment:
Would update last line to be "These innovations enable every transcript that is captured from individual cells to be mapped to its cognate cell and counted as a single transcript, correcting for PCR bias with UMIs."

Technically batch effect correction happens downstream.